test/DICOMSeriesTest.js

```diff
@@ -70,7 +70,8 @@ test('Test reading a DICOM file with the synchronous API', t => {
 test('Test reading a DICOM file with the synchronous API, assume sorted', t => {
   const image = readImageLocalDICOMFileSeriesSync(testFiles.sort(), true)
   t.is(image.imageType.dimension, 3, 'dimension')
-  t.is(image.imageType.componentType, IntTypes.UInt8, 'componentType')
+  t.is(image.imageType.componentType, IntTypes.Int16, 'componentType')
+  t.is(image.imageType.pixelType, PixelTypes.Scalar, 'pixelType')
```

Comment:
duplicated line `t.is(image.imageType.pixelType, PixelTypes.Scalar, 'pixelType')`

Reply:
Good catch and thanks for the review! Removed here: https://github.com/InsightSoftwareConsortium/itk-js/pull/322